src/server/transaction/server/driver.go

```diff
@@ -168,6 +168,8 @@ func (d *driver) runTransaction(txnCtx *txncontext.TransactionContext, info *tra
 			err = directTxn.DeleteBranch(request.DeleteBranch)
 		} else if request.UpdateJobState != nil {
 			err = directTxn.UpdateJobState(request.UpdateJobState)
+		} else if request.StopJob != nil {
+			err = directTxn.StopJob(request.StopJob)
```

Comment:
`StopJob` wasn't useable in a transaction because this was missing.

Reply:
Oh hmm, do we want `stop job` in general to use the current transaction?

Reply:
Oh, that's a good point, but it comes up against the same thing that makes `pachctl start commit` with multiple branches tough to do in a transaction - I'll handle that in the PR for that feature.